experimental/CliffordOrders/src/CliffordOrders.jl

```diff
@@ -0,0 +1,1056 @@
+    provided and the parent object of the element provided are not identical."
+    return C(coefficients(elt))
+end
+
+################################################################################
+#
+#  Containment
+#
+################################################################################
+
+@doc raw"""
+    in(x::CliffordAlgebraElem, C::CliffordOrder)
+
+Return true if the element $x$ is contained in the Clifford order $C$.
+"""
+function Base.in(x::CliffordAlgebraElem, C::CliffordOrder)
+  if !(ambient_algebra(C) === parent(x))
+    return false
+  end
+  coe, coeids = coefficients(x), coefficient_ideals(C)
+  for i in 1:rank(C)
+    if !(coe[i] in coeids[i])
+      return false
+    end
+  end
+  return true 
+end
+
+### ZZ ###
+
+@doc raw"""
+    in(x::CliffordAlgebraElem, C::ZZCliffordOrder)
+
+Return true if the element $x$ is contained in the Clifford order $C$.
+"""
+function Base.in(x::CliffordAlgebraElem, C::ZZCliffordOrder)
+  if !(ambient_algebra(C) === parent(x))
+    return false
+  end
+  coe = coefficients(x)
+  for i in 1:rank(C)
+    if !(is_integral(coe[i]))
+      return false
+    end
+  end
+  true 
+end
+
+################################################################################
+#
+#  Basic field access
+#
+################################################################################
+
+##### Order  #####
+@doc raw"""
+    base_ring(C::CliffordOrder) -> Ring
+
+Return the base ring of the Clifford order $C$.
+"""
+base_ring(C::CliffordOrder) = C.base_ring
+
+@doc raw"""
+    ambient_algebra(C::CliffordOrder) -> CliffordAlgebra
+
+Return the Clifford algebra of the ambient space of the underlying lattice of $C$.
+"""
+ambient_algebra(C::CliffordOrder) = C.ambient_algebra
+
+@doc raw"""
+    rank(C::CliffordOrder) -> Int
+
+Return the rank of the Clifford order $C$ over its base ring.
+"""
+rank(C::CliffordOrder) = C.rank
+
+@doc raw"""
+    lattice(C::CliffordOrder) -> QuadLat
+
+Return the underlying even quadratic lattice of the Clifford order $C$.
+"""
+lattice(C::CliffordOrder) = C.lattice
+
+@doc raw"""
+    gram_matrix(C::CliffordOrder) -> MatElem
+
+Return the Gram matrix with respect to the fixed pseudo-basis of the Clifford order $C$.
+"""
+gram_matrix(C::CliffordOrder) = C.gram
+
+@doc raw"""
+    coefficient_ideals(C::CliffordOrder) -> Vector{NumFieldOrderFractionalIdeal}
+
+Return the vector of coefficient ideals of the canonical pseudo-basis of $C$.
+"""
+coefficient_ideals(C::CliffordOrder) = C.coefficient_ideals
+
+### ZZ ###
+
+@doc raw"""
+    base_ring(C::ZZCliffordOrder) -> ZZRing
+
+Return the base ring of the Clifford order $C$.
+"""
+base_ring(C::ZZCliffordOrder) = C.base_ring
+
+@doc raw"""
+    ambient_algebra(C::ZZCliffordOrder) -> CliffordAlgebra
+
+Return the Clifford algebra of the ambient space of the underlying lattice of $C$.
+"""
+ambient_algebra(C::ZZCliffordOrder) = C.ambient_algebra
```

Comment:
I think for the other orders in Oscar, this is just called `algebra`.

Reply:
Changed in this [commit](https://github.com/oscar-system/Oscar.jl/pull/4370/commits/d4b78c05016fbec7d6ab36f6b18b4b8bec2d084d)